security/aws/rds_unencrypted/README.md

```diff
@@ -0,0 +1,62 @@
+## AWS Unencrypted RDS Instances
+ 
+### What it does
+
+This policy checks all Relational Database Service (RDS) instances and reports on any that are unencrypted. When such a RDS instance is detected, the user can choose to delete by enabling 'delete action' option as mentioned in Enable delete action section below.
+
+### Functional Details
+ 
+When a Unencrypted RDS instance is detected, an email action is triggered automatically to notify the specified users of the incident. Users then have the option to delete the RDS instance after manual approval if needed. 
+ 
+#### Input Parameters
+ 
+- *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
+- *Ignore tags* - RDS instances with any of these tags will be ignored 
+ 
+### Required RightScale Roles
+ 
+- policy_manager
+- admin or credential_viewer
+
+### Enable delete action
+
+Perform below steps to enable delete action.
+
+- *Edit the file [AWS_Unencrypted_RDS_Instances](https://github.com/rightscale/policy_templates/tree/master/security/aws/rds_unencrypted/AWS_Unencrypted_RDS_Instances.pt)
+- *uncomment the line which contains 'escalate $delete_unEncripted_RDS_instances_approval' and save the changes.
+- *upload the modified file and apply the policy.
+
+Note: 
+- *RDS Instances with 'DB Instance Status' other than 'Available' and RDS instances with 'delete Protection enabled' cannot be deleted
+- *When delete action is performed, DB snapshot gets created with name '<--DB_Instance_Identifier-->-finalSnapshot' Ex mySQL-DBinstance-finalSnapshot before deleting DB instance.
+- *For Aurora instance, policy creates Cluster snapshot Since DB instance snapshot cannot be created directly.
+
+### AWS Required Permissions
+
+This policy requires permissions to describe AWS Unencrypted RDS instances, tags and Delete RDS instances.
```

Comment:
First sentence -> This policy requires permissions to describe AWS Unencrypted RDS instances, describe RDS tags, and delete RDS instances.

Reply:
changed it to "This policy requires permissions to describe AWS Unencrypted RDS instances, describe RDS tags, create DB cluster snapshot, describe DB cluster snapshot and delete RDS instances."